### PR TITLE
Gender lang fix2

### PR DESCRIPTION
**Pull request by richierocks**
I think this is the minimal fix for the gender-related language.

### CODE REVIEW

File: chapter4.Rmd [hunk 1]
```diff
@@ -418,7 +412,7 @@ test_object("factor_survey_vector", eq_condition = "equal", undefined_msg = msg,
 test_object("male", undefined_msg = msg, incorrect_msg = msg)
 test_object("female", undefined_msg = msg, incorrect_msg = msg)
 test_output_contains("male > female", incorrect_msg = msg)
-success_msg("Phew, it seems that R is gender-neutral. Or maybe it just wants to stay out of these discussions ;-).")
+success_msg("Phew, it seems that R is not a sexist language.")
```
Comment (ismayc) [inline chapter4.Rmd]: I think I'd make this just a standard "Good job." `success_msg`.

File: chapter4.Rmd [hunk 2]
```diff
@@ -19,7 +19,7 @@ The term factor refers to a statistical data type used to store categorical vari
 
 It is important that R knows whether it is dealing with a continuous or a categorical variable, as the statistical models you will develop in the future treat both types differently. (You will see later why this is the case.)
 
-A good example of a categorical variable is the variable 'Gender'. A human individual can either be "Male" or "Female", making abstraction of inter-sexes. So here "Male" and "Female" are, in a simplified sense, the two values of the categorical variable "Gender", and every observation can be assigned to either the value "Male" of "Female".
+A good example of a categorical variable is sex. In many circumstances you can limit the sex categories to "Male" or "Female". (Sometimes you may need different categories. For example, you may need to consider chromosomal variation or hermaphroditic animals, but you will always have a finite number of categories.)
```
Comment (gvwilson) [inline chapter4.Rmd]: "…you may need to consider chromosomal variation, hermaphroditic animals, or different cultural norms, but you will always…"

File: chapter4.Rmd [hunk 3]
```diff
@@ -214,7 +208,7 @@ When you first get a data set, you will often notice that it contains factors wi
 levels(factor_vector) <- c("name1", "name2",...)
 ```
 
-A good illustration is the raw data that is provided to you by a survey. A standard question for every questionnaire is the gender of the respondent. You remember from the previous question that this is a factor and when performing the questionnaire on the streets its levels are often coded as `"M"` and `"F"`.
+A good illustration is the raw data that is provided to you by a survey. A standard question for every questionnaire is the sex of the respondent. You remember from the previous question that this is a factor and when performing the questionnaire on the streets its levels are often coded as `"M"` and `"F"`.
```
Comment (gvwilson) [inline chapter4.Rmd]: "A common question is the sex of the respondent, which is still often over-simplified to `"M"` and `"F"`. However many categories there are, this is a factor."

File: chapter4.Rmd [hunk 4]
```diff
@@ -363,10 +357,10 @@ skills: 1
 key: 90ecc160d1ebf2f75bf53f9c3843fc1632bdd0a5
 ```
 
-In `factor_survey_vector` we have a factor with two levels: Male and Female. But how does R value these relatively to each other? In other words, who does R think is better, males or females?
+You might wonder what happens when you try to compare elements of a factor.  In `factor_survey_vector` you have a factor with two levels: `"Male"` and `"Female"`. But how does R value these relatively to each other? In other words, who does R think is greater, males or females?
```
Comment (gvwilson) [inline chapter4.Rmd]: "In other words, which does R think should come first in sorted data, males or females?"

File: chapter4.Rmd (hunk 1 above)
Comment (gvwilson) [reply chapter4.Rmd]: :+1:

Comment (richierocks) [reply chapter4.Rmd]: Updated via https://github.com/datacamp/courses-intro-to-r/commit/be438ff3ad1dd9bd9f9cbd761000a4a676bda0a9

File: chapter4.Rmd (hunk 2 above)
Comment (richierocks) [reply chapter4.Rmd]: Fixed via https://github.com/datacamp/courses-intro-to-r/commit/7ae518f8323dbe30bbf8624c1e9b24a2c614d85c

File: chapter4.Rmd (hunk 3 above)
Comment (richierocks) [reply chapter4.Rmd]: Updated via https://github.com/datacamp/courses-intro-to-r/commit/a36c7926c7c68ba180be569535f23fbb893ac835

File: chapter4.Rmd (hunk 4 above)
Comment (richierocks) [reply chapter4.Rmd]: "Greater" is used because the question is about using then greater than operator on factor values. So I think I'm happy with the sentence as it is.

Comment (justmytwospence) [reply chapter4.Rmd]: Agree with @gvwilson that this phrasing is fraught. Maybe rephrase as:

In other words, which level does R consider to be greater, `male` or `female`?

Therfore, we avoid using the word "value", and also by using inline code for `male` and `female` it is clear that we aren't referring to men and women in general.

**Comment by ismayc** [comment]
@yashasroy Any updates here?

**Comment by ismayc** [comment]
@yashasroy Never mind. Just saw your message in the other pull request.

File: chapter4.Rmd (hunk 4 above)
Comment (ncarchedi) [inline chapter4.Rmd]: "relatively" --> "relative". Also, why not just cut the last sentence? It's annoying and unnecessary.

**Comment by yashasroy** [comment]
@ncarchedi Assignment text has been edited.
